content/docs/deployment/csmoperator/modules/authorization.md

```diff
@@ -144,3 +144,78 @@ Follow the instructions available in CSM Authorization for [Configuring the CSM
 ### Configure a Dell CSI Driver with CSM Authorization
 
 Follow the instructions available in CSM Authorization for [Configuring a Dell CSI Driver with CSM for Authorization](../../../helm/modules/installation/authorization/#configuring-a-dell-csi-driver-with-csm-for-authorization).
+
+## Upgrading the Authorization Proxy Server
+
+1. Modifying the existing Authorization Proxy Server installation directly via `kubectl edit`
+  
+   ```sh
+
+   kubectl get csm -n <module-namespace>
+   ```
+
+   For example - If the Authorization Proxy Server is installed in authorization namespace then run this command to get the object name
+
+   ```sh
+    
+   kubectl get csm -n authorization
+   ```
+
+   use the object name in `kubectl edit` command.
+
+   ```sh
+
+   kubectl edit csm <object-name> -n <module-namespace>
+   ```
+
+   For example - If the object name is authorization then use the name as authorization and if the namespace is authorization, then run this command to edit the object
+
+   ```sh
+
+   kubectl edit csm authorization -n authorization
+   ```
+
+2. Modify the installation
+
+   - Update the CSM Authorization Proxy Server configVersion  
+   - Update the images for proxyService, tenantService, roleService and storageService
+
+
+## Upgrading CSI Driver, Authorization sidecar with Authorization module enabled
+
+1. Modifying the existing driver and module installation directly via `kubectl edit`
+
+   ```sh
+
+   kubectl get <driver-object> -n <driver-namespace>
+   ```
+
+   For example - If the CSI PowerFlex driver is installed in vxflexos namepace then run this command to get the object name
+
+   ```sh
+
+   kubectl get csm -n vxflexos
+   ```
+
+   use the object name in `kubectl edit` command.
+
+   ```sh
+
+   kubectl edit csm <driver-object>/<object-name> -n <driver-namespace>
```

Comment:
Same as above.

Reply:
I have given example in below command
For example - If the object name is vxflexos and the driver is installed in vxflexos namespace, then run this command to edit the object
  kubectl edit csm vxflexos -n vxflexos